CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# dbt_google_ads_source v0.7.0
+## 🚨 Breaking Changes 🚨
+- The `api_source` variable is now defaulted to `google_ads` as opposed to `adwords`. The Adwords API has since been depricated by Google and is now no longer the standard API for the Google Ads connector. Please ensure you are using a Google Ads API version of the Fivetran connector before upgrading this package. ([#28](https://github.com/fivetran/dbt_google_ads_source/pull/28))
+  - Please note, the `adwords` version of this package will be fully removed from the package in August of 2022.
```

Comment:
I think it might be helpful to specify what this exactly entails -- I figure it can't hurt to be explicit. 

Maybe something along the lines of "Please note, the `adwords` version of this package will be fully removed and every model under `models/adswords_connector` will be removed in favor of `models/google_ads_connector` models"

Reply:
That is a great point. Will update!